### DISCUSSION TITLE
advice for setup project

### DISCUSSION THREAD

**Discussion by marioveyna** [taligentx/dscKeybusInterface#179] [dscKeybusInterface Discussion]
i know this forum is not for this, but need help to find some `help forum` or reddit thread for help setup this project, i not a electronic expert and a year ago tried to do it with pushbullet but didn't work, and i drop keep trying... can anyone recomend me where get some help? thanks

**Comment by kricon** [comment]
What panel do you have? Are you sure your wiring is correct (tried `Status`/`KeybusReader `sketch)?
Explain a bit better _tried to do it with pushbullet but didn't work_ what exactly you tried and what didnt work?

**Reply by marioveyna** [reply]
thanks for your reply, i have a pc1555mx panel, using a esp8266 with some dupont wires and a tiny protoboard, also use the pushbullet.ino file and wire like the diagram inside, i get some var type error and correct this but seems to works, got connected message on var but no triggers or pushbullet push notification.. :/

**Reply by kricon** [reply]
> ...using a esp8266 with some **dupont wires** and a tiny **protoboard**, also use the pushbullet.ino file and wire like the diagram inside...

This could be the issue with wiring. Its even noted in readme: `Connections should be soldered, breadboards can cause issues.` I strongly advise putting proto/breadboard out of equation, especially if it is low quality one, and make firm connections. Even twisting the wires to resistors without soldering are better than using cheap breadboard.

Which relase of _pushbullet.ino_ do you use, revision v1.1 or v1.3? V1.1 included in `master` branch have wrong resistor values written for ESP8266 - it should be 33k/10k instead of 15k/10k ones because of possibility to fry ESP8266 inputs with too high voltage (ESP operates at 3.3V - Arduino at 5V).

I suppose you put correct wifi ssid/pass and also pushbullet token values between quotes in sketch? Im checking, just to be sure that part is correct.

Can you check your resistors to verify they are 33k/10k ohms and flash `keybusreader` to verify that wiring is correct? Just open terminal after flashing ESP and connect it to PC1555MX panel (it can be connected to the panel even while flashing) and see if you get any output from panel.

**Reply by marioveyna** [reply]
thank you for your complete reply!! i check all this weekend, try to solder it and check terminal and try to post here, thanks again so much! keep informed ;)
